server/src/main/scala/ch.epfl.scala.index.server/routes/package.scala

```diff
@@ -10,29 +10,33 @@ package object routes {
       (
         "q" ? "*",
         "page".as[Int] ? 1,
+        "total".as[Int] ? SearchParams.resultsPerPage,
```

Comment:
Could we also include information about available pages? Currently one needs to call next page until the response is empty (but this is probably not in the scope of this PR).

Reply:
I have added the `currentPage` and `pageTotal` fields: https://index-dev.scala-lang.org/api/search?q=*&scalaVersions=scala3&sort=stars&total=20&page=2